java/cuvs-java/pom.xml

```diff
@@ -21,8 +21,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.nvidia.cuvs</groupId>
     <artifactId>cuvs-java</artifactId>
-    <!-- NOTE: The version automatically gets updated when ci/release/update-version.sh is invoked. -->
-    <!--CUVS_JAVA#VERSION_UPDATE_MARKER_START--><version>25.08.0</version><!--CUVS_JAVA#VERSION_UPDATE_MARKER_END-->
+    <version>${version}</version>
```

Comment:
I thought I was supportive of this change until I got to this part of the change.  I don't think it's a good idea not to have a concrete version number in the project pom.

For one, every `mvn` command run by developers would now have to be suffixed with a `-Dversion=../get-version.sh`.

For another, this `pom.xml` accompanies the project artifact at the time of publication to Maven Central.  That would need the explicit version in the `pom.xml` and in the artifact jar.